src/components/CardSchemes/components/PaymentMethodIcon/PaymentMethodIcon.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { keys } from 'lodash';
+
+import { sizes } from '../../../../styles/constants';
+// eslint-disable-next-line max-len
+import schemeMap from '../../../CreditCardDetails/components/scheme-icons/card-scheme-icons';
+
+const { BYTE, KILO, MEGA, GIGA } = sizes;
+
+const PaymentMethodIconWrapBaseStyles = ({ theme, size }) => css`
+  height: ${theme.iconSizes[size]};
+  width: auto;
+`;
+const PaymentMethodIconWrap = styled('div')(PaymentMethodIconWrapBaseStyles);
+/**
+ * A single card scheme icon
+ */
+const PaymentMethodIcon = ({ schemeId, size }) => {
+  const IconSvg = schemeMap[schemeId];
+
+  if (!IconSvg) {
+    return null;
+  }
+
+  return (
+    <PaymentMethodIconWrap size={size}>
+      <IconSvg
+        className={css`
```

Comment:
Instead you should just use the `css` prop, which will be transpiled by the emotion babel plugin.